pkg/lib/dialogs.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* DIALOG PRESENTATION PROTOCOL
+ *
+ * Example:
+ *
+ * import { WithDialogs, useDialogs } from "dialogs.jsx";
+ *
+ * const App = () =>
+ *   <WithDialogs>
+ *     <Page>
+ *       <ExampleButton />
+ *     </Page>
+ *   </WithDialogs>;
+ *
+ * const ExampleButton = () => {
+ *   const D = useDialogs();
```

Comment:
IMHO the example/documentation should use the same "best practice" name as the code, i.e. `Dialogs`.

Reply:
Of course, ouch.

Reply:
Unresolving, this wasn't actually changed. Forgot `git add` or something?